src/test/java/org/mitre/synthea/world/agents/PayerTest.java

```diff
@@ -528,6 +528,7 @@ public void payerMemberYears() {
     for (int year = 0; year <= 55; year++) {
       ((Map<Integer, Double>) person.attributes.get("QOL")).put(2000 + year, 1.0);
       long currentTime = startTime + Utilities.convertTime("years", year);
+      currentTime = currentTime - (year * 60 * 60 * 6 * 1000);
```

Comment:
I think this is wrong.

- Line 530 says "Set current time to start time plus N years."
- Line 531 says "Set current time to current time minus N years."

Current time is always the start time.